workflow-templates/psake-ci.yml

```diff
@@ -37,56 +26,51 @@ jobs:
-        psake ci
+      - name: Run CI task using psake
+        run: |
+          $env:OCTOPUS_DEPLOY_APIKEY = "${{ secrets.OCTOPUS_DEPLOY_API_KEY }}"
+          psake ci
 
-    ## Uncomment the next two steps if you want to create an artifact
-    ## If a git tag was not being pushed, then we create an artifact
-    ## that contains the built packages
-    ## ref: https://wiki.meniga.net/display/DEVOPS/Migrate+build+process+to+GitHub+Actions+using+Windows+self-hosted+runner#MigratebuildprocesstoGitHubActionsusingWindowsself-hostedrunner-Moreadvancedbuildsteps
-    # - name: Add git tag to environment
-    #   if: startsWith(github.ref, 'refs/tags') == 'false'
-    #   id: set-packageversion
-    #   run: |
-    #     psake Set-PackageVersionOutput
+      ## Uncomment the next two steps if you want to create an artifact
+      ## If a git tag was not being pushed, then we create an artifact
+      ## that contains the built packages
+      ## ref: https://wiki.meniga.net/display/DEVOPS/Migrate+build+process+to+GitHub+Actions+using+Windows+self-hosted+runner#MigratebuildprocesstoGitHubActionsusingWindowsself-hostedrunner-Moreadvancedbuildsteps
+      # - name: Add git tag to environment
+      #   if: startsWith(github.ref, 'refs/tags') == 'false'
+      #   id: set-packageversion
+      #   run: |
+      #     psake Set-PackageVersionOutput
 
-    ## Upload the artifact so that it is accessible from the workflow summary page
-    # - name: Upload artifact
-    #   if: startsWith(github.ref, 'refs/tags') == 'false'
-    #   uses: actions/upload-artifact@v2
-    #   with:
-    #     name: ${{ env.PROJECT_NAME }}.${{ steps.set-packageversion.outputs.packageVersion }}
-    #     path: build/Latest
-    #     retention-days: 10
+      # - name: Zip release
+      #   run: |
+      #     7za a -tzip "${{ steps.set-packageversion.outputs.packageVersion }}.zip" build/Latest
 
-    # Ideally, you would supply the action with a body that
-    # further explains the changes in the release
-    # For more information on how to customize the release
-    # ref: https://github.com/softprops/action-gh-release#-customizing
-    - name: Create release (if tag was pushed)
-      if: startsWith(github.ref, 'refs/tags')
-      uses: softprops/action-gh-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        draft: false
-        prerelease: false
-        # Adjust as needed, should point to the path(s) of the build output
-        # of the CI step
-        # Newline-delimited globs of paths to assets to upload for release
-        files: build/Latest
+      # Ideally, you would supply the action with a body that
+      # further explains the changes in the release
+      # For more information on how to customize the release
+      # ref: https://github.com/softprops/action-gh-release#-customizing
+      - name: Create release (if tag was pushed)
+        if: startsWith(github.ref, 'refs/tags')
```

Comment:
Also note the `if` statement here. We only create a GitHub release if a tag is pushed

Reply:
Please see my comment